extras/rebalance/directory-rebalance.py

```diff
@@ -8,18 +8,18 @@
 import hashlib
 import logging
 
-def size_fmt(num, suffix='B'):
-    for unit in ['','Ki','Mi','Gi','Ti','Pi','Ei','Zi']:
+def size_fmt(num):
+    for unit in ['B','KiB','MiB','GiB','TiB','PiB','EiB','ZiB']:
         if abs(num) < 1024.0:
-            return "{:3.1f} {}{}".format(num, unit, suffix)
+            return f"{num:4.2f} {unit}"
```

Comment:
I'm not sure if you want to align the numbers. Which output do you want exactly ?

```python
>>> for i in [1, 10, 100, 1000]:
...     print(f'{i:4.2f}')
...
1.00
10.00
100.00
1000.00
>>> for i in [1, 10, 100, 1000]:
...     print(f'{i:7.2f}')
...
   1.00
  10.00
 100.00
1000.00
```